lib/generators/shopify_app/shop_model/templates/shop_helper.rb

```diff
@@ -0,0 +1,9 @@
+
+  protected
+
+  def shop
+    @shop ||= Shop.find(session[:shopify])
```

Comment:
How about this for a more concise approach? It'll return nil if not found and you can drop the rescue block.
```ruby
@shop ||= Shop.find_by(id: session[:shopify])
```